app/entry.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import {render} from 'react-dom'
+import App from './App.jsx'
```

Comment:
JS module imports do not require the .js or .jsx extensions

Reply:
This needs an update in webpack configuration to resolve file extensions